CONTRIBUTING.md

```diff
@@ -19,10 +19,10 @@ New features should, in general, be added to the master branch.
 
 ### Code Style
 
-Codestyle TBD.
+This project uses the cassandra code style which is based on Sun’s Java coding conventions. Formatting rules for eclipse can be found [here](https://github.com/Ericsson/ecchronos/blob/ecchronos-1.0/code_style.xml).
```

Comment:
Once sentence per line?

I don't think the link works like that. You may have to use rawgit. Checkout this example that I did for the performance charts in ecAudit. https://github.com/Ericsson/ecaudit/blob/release/c2.2/doc/description.md#performance

Or pehaps it is fine to just say that the file can be found at the root of the repo?

Reply:
I'll try to add a relative path and see if that works.